doc/source/whatsnew/v0.19.0.txt

```diff
@@ -436,6 +438,15 @@ Deprecations
 - top-level ``pd.ordered_merge()`` has been renamed to ``pd.merge_ordered()`` and the original name will be removed in a future version (:issue:`13358`)
 - ``Timestamp.offset`` property (and named arg in the constructor), has been deprecated in favor of ``freq`` (:issue:`12160`)
 
+.. _whatsnew_0190.deprecations.read_csv_arg_rename:
+
+All multi-word arguments to ``pd.read_csv()`` will now have their individual words separated by an underscore, and their
+ underscore-less versions have not be deprecated (:issue:`13386`). They are:
+
+- The ``chunksize``, ``dayfirst``, ``doublequote``, ``escapechar``, ``lineterminator``, ``quotechar``, ``skipinitialspace``,
```

Comment:
For future reference, the example you are referring to is <a href="https://github.com/pydata/pandas/blob/master/doc/source/whatsnew/v0.18.0.txt#L614">here</a>.  Ah, got it.  Done.